src/state_machine/wrapping.py

```diff
@@ -0,0 +1,1063 @@
+#!/usr/bin/env python
+
+import copy
+
+from sensor_msgs.msg import CameraInfo, Image
+# from std_msgs.msg import Int32
```

Comment:
no obsolete imports please. this goes for the whole file.

Reply:
yap fixed